integration/build_test.go

```diff
@@ -32,96 +31,80 @@ import (
 	"github.com/okteto/okteto/pkg/registry"
 )
 
-const (
-	buildGitRepo   = "git@github.com:okteto/react-getting-started.git"
-	buildGitFolder = "react-getting-started"
-	buildManifest  = "okteto.yml"
-)
-
 func TestBuildCommand(t *testing.T) {
 
 	ctx := context.Background()
 	oktetoPath, err := getOktetoPath(ctx)
 	if err != nil {
 		t.Fatal(err)
 	}
+	const (
+		gitRepo          = "git@github.com:okteto/go-getting-started.git"
+		repoDir          = "go-getting-started"
+		manifestFilename = "okteto.yml"
+		manifestContent  = `build:
+  app:
+    context: .`
+	)
+
+	var (
+		testID           = strings.ToLower(fmt.Sprintf("TestBuildCommand-%s-%d", runtime.GOOS, time.Now().Unix()))
+		testNamespace    = fmt.Sprintf("%s-%s", testID, user)
+		originNamespace  = getCurrentNamespace()
+		expectedImageTag = fmt.Sprintf("%s/%s/%s:dev", okteto.Context().Registry, testNamespace, "app")
+	)
+
+	if err := createNamespace(ctx, oktetoPath, testNamespace); err != nil {
```

Comment:
I'd move this function call after all the setup. So in case it fails (cloning the repo or writing the file) it doesn't create an empty namespace